examples/altonegen.c

```diff
@@ -44,7 +44,9 @@
 
 #include "common/alhelpers.h"
 
-#include "win_main_utf8.h"
+#ifdef _WIN32
+# include "win_main_utf8.h"
+#endif
```

Comment:
Is there a reason this was put behind an `ifdef`, when the header already checks for Windows?

Reply:
I made a mistake compiling the examples, leaving out an include directory path.  But for me I prefer having the logic that include or excludes platform specific files where I can see it rather than going to the header to check. So my steps went something like, “compilation on Mac failed, why? Saw the Windows specific include, ifdef’ed it out, compilation succeeded. Link failed with code in alhelpers.c missing.  Found alhelpers, added the directory, compiled and linked altonegen.  Didn’t go back to undo ifdefs cuz the code feels more readable that way.

I shall revert if if you prefer.